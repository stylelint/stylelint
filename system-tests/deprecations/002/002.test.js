/* @flow */
"use strict"

const systemTestUtils = require("../../systemTestUtils")
const stylelint = require("../../../lib")

it("deprecations/002", () => {
  return stylelint.lint({
    files: [systemTestUtils.caseStylesheetGlob("deprecations/002")],
    configFile: systemTestUtils.caseConfig("deprecations/002"),
  }).then((output) => {
    expect(systemTestUtils.prepResults(output.results)).toMatchSnapshot()
  })
})
