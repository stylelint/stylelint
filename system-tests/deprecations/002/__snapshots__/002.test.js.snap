// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`deprecations/002 1`] = `
Array [
  Object {
    "deprecations": Array [
      Object {
        "reference": "https://stylelint.io/user-guide/rules/at-rule-empty-line-before/",
        "text": "'at-rule-empty-line-before's' \\"all-nested\\" option has been deprecated and in 8.0 will be removed. Instead use the \\"inside-block\\" option.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/at-rule-empty-line-before/",
        "text": "'at-rule-empty-line-before's' \\"blockless-group\\" option has been deprecated and in 8.0 will be removed. Instead use the \\"blockless-after-blockless\\" option.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/comment-empty-line-before/",
        "text": "'comment-empty-line-before's' \\"between-comments\\" option has been deprecated and in 8.0 will be removed. Instead use the \\"after-comment\\" option.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/max-nesting-depth/",
        "text": "'max-nesting-depth's' \\"at-rules-without-declaration-blocks\\" option has been deprecated and in 8.0 will be removed. Instead use the \\"blockless-at-rules\\" option.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/rule-nested-empty-line-before/",
        "text": "'rule-nested-empty-line-before' has been deprecated and in 8.0 will be removed. Instead use 'rule-empty-line-before'.",
      },
    ],
    "errored": true,
    "ignored": undefined,
    "invalidOptionWarnings": Array [],
    "source": "deprecations/002/stylesheet.css",
    "warnings": Array [
      Object {
        "column": 3,
        "line": 2,
        "rule": "at-rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before at-rule (at-rule-empty-line-before)",
      },
      Object {
        "column": 3,
        "line": 4,
        "rule": "at-rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before at-rule (at-rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 7,
        "rule": "at-rule-empty-line-before",
        "severity": "error",
        "text": "Unexpected empty line before at-rule (at-rule-empty-line-before)",
      },
      Object {
        "column": 3,
        "line": 11,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Unexpected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 3,
        "line": 24,
        "rule": "custom-property-empty-line-before",
        "severity": "error",
        "text": "Unexpected empty line before custom property (custom-property-empty-line-before)",
      },
      Object {
        "column": 3,
        "line": 31,
        "rule": "declaration-empty-line-before",
        "severity": "error",
        "text": "Unexpected empty line before declaration (declaration-empty-line-before)",
      },
      Object {
        "column": 3,
        "line": 37,
        "rule": "rule-nested-empty-line-before",
        "severity": "error",
        "text": "Unexpected empty line before nested rule (rule-nested-empty-line-before)",
      },
    ],
  },
]
`;
